frontend/src/state/data-params.ts

```diff
@@ -21,6 +22,7 @@ export type DataParamParam = Readonly<{
 
 export const dataParamConfig: Record<string, DataParamGroupConfig> = {
   ...HAZARD_DOMAINS,
+  ...RISK_DOMAINS,
```

Comment:
This overwrites hazard params, when `RISK_DOMAINS` keys are the same as `HAZARD_DOMAINS` keys.

Reply:
For now, I've simplified `RISKS_DOMAINS` to a single `risks` parameter group.